src/lib/modules/blockchain_process/blockchainProcessLauncher.js

```diff
@@ -66,6 +67,16 @@ class BlockchainProcessLauncher {
     this.events.on('exit', () => {
       this.blockchainProcess.send('exit');
     });
+
+    this.embark.registerAPICall(
+      'get',
+      '/embark-api/regular-txs',
+      (req, _res) => {
+        if(this.isDev && req.query.mode === 'on'){
+          this.blockchainProcess.send({action: constants.blockchain.initDevFunds});
```

Comment:
This will only work in the case where the blockchain node is in the same terminal (subprocess). 
We'd need to communicate to the external process too via IPC or something that it should start.
Also, FYI, I added the ability for subprocesses to listen to events (an abstraction of the `process.send` method) where the blockchain process can do `this.events.on('regularTxs:start');` and it will work when anywhere in Embark we do `this.events.emit('regularTxs:start');`

Reply:
@jrainville - this has been updated to support a standalone blockchain process. I've also moved the commands in support of the event-based abstration on the subprocess to what I believe is what you referring to.

In addition, I've added start/stop console commands `regularTxs on/off`.